packages/react-devtools-shared/src/backend/console.js

```diff
@@ -209,7 +224,21 @@ export function patch({
           debugger;
         }
 
-        originalMethod(...args);
+        if (isInStrictMode) {
+          if (!consoleSettingsRef.hideDoubleLogsInStrictMode) {
+            // TODO(luna) pick a better color than gray
+            if (typeof window !== 'undefined') {
```

Comment:
Seems like this might get some false positives for environments (like Jest tests) where `window` might be mocked.

Maybe we could do something like this instead?

```js
let isNode = false;
try {
  isNode = this===global;
} catch (error) {}
```

We'd only need to do this once (during module initialization) then we could just check the `isNode` value in this function.